web-ui/src/app/components/shared/shared.module.ts

```diff
@@ -0,0 +1,17 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { TickerPanelsComponent } from "./ticker-panels/ticker-panels.component";
+
+@NgModule({
+  imports: [
+    CommonModule
+  ],
+  declarations: [
+    TickerPanelsComponent
+  ],
+  exports: [
+    CommonModule,
+    TickerPanelsComponent
+  ]
+})
+export class SharedModule { }
```

Comment:
I'm not familiar on creating most than a module on Angular, could you please describe why you did it?

Reply:
According to the course that I took on pluralsight [Angular Getting Started](https://app.pluralsight.com/player?course=angular-2-getting-started-update&author=deborah-kurata&name=angular-2-getting-started-update-m12&clip=0&mode=live), it is a good practice to de-couple your app into several modules for better code organization and refactor friendly.

[This forum](https://stackoverflow.com/questions/39756178/one-single-ngmodule-versus-multiples-ones-with-angular-2?utm_medium=organic&utm_source=google_rich_qa&utm_campaign=google_rich_qa) also explains the benefits.

And here its an example of multi modules:
![image](https://user-images.githubusercontent.com/5649650/38752440-31242a56-3f29-11e8-98bb-3cf0d2fbc3f0.png)